crates/nu-command/src/filters/find.rs

```diff
@@ -535,6 +488,59 @@ fn find_with_rest_and_highlight(
     }
 }
 
+fn terms_record_formatter(
+    terms: Vec<Value>,
+    config: Config,
+    ls_colors: LsColors,
+    string_style: Style,
+) -> ValueFormatter {
+    ValueFormatter::from_fn(move |value| {
+        let (cols, mut vals, span) = match value {
+            Value::Record { cols, vals, span } => (cols, vals, span),
+            _ => return value,
+        };
+
+        'vals: for val in &mut vals {
+            let val_str = val.into_string("", &config);
+            let lower_val = val.into_string("", &config).to_lowercase();
+
+            for term in terms.iter() {
+                let term_str = term.into_string("", &config);
+                let lower_term = term.into_string("", &config).to_lowercase();
+
+                if lower_val.contains(&lower_term) {
+                    let val_str = if config.use_ls_colors {
+                        // Get the original LS_COLORS color
```

Comment:
if we've made the decision here to `use_ls_colors`, that infers to me that we know this is the output of `ls`. So, perhaps we should add hyperlinking here too. Or better yet, maybe put that code in nu-utils and just reference it from there. We just don't want any value type things I nu-utils so that it can be referenced from any crate.